packages/augment-api/tsconfig.build.json

```diff
@@ -4,7 +4,18 @@
     "module": "CommonJS",
     "outDir": "./lib/cjs",
     "skipLibCheck": true,
-    "noUnusedLocals": false
+    "noUnusedLocals": false,
+    "paths": {
+      "@kiltprotocol/augment-api/*": [
+        "./src/*"
+      ],
```

Comment:
Yeah that's the trickery I was talking about... have you tried adding `"@kiltprotocol/augment-api/interfaces/*"` as well to deal with the import issues?

Reply:
Technically `@kiltprotocol/augment-api/*` also catches any subpaths. And VScode stopped complaining with this tweak, but the build process from the project root was still failing. Maybe this is something that should rather be declared in the root?

Reply:
I imagine so, bc it could just fail then in later build steps

Reply:
Tried it, and it did not work. Maybe we can get back to it once you are also back.